cmd/process-agent/collector.go

```diff
@@ -156,10 +210,12 @@ func (l *Collector) run(exit chan bool) error {
 
 	processForwarderOpts := forwarder.NewOptions(keysPerDomains(l.cfg.APIEndpoints))
 	processForwarderOpts.DisableAPIKeyChecking = true
+	processForwarderOpts.RetryQueueSize = l.cfg.QueueSize // Allow more in-flight requests than the default
 	processForwarder := forwarder.NewDefaultForwarder(processForwarderOpts)
 
 	podForwarderOpts := forwarder.NewOptions(keysPerDomains(l.cfg.OrchestratorEndpoints))
-	podForwarderOpts.DisableAPIKeyChecking = true
+	podForwarderOpts.DisableAPIKeyChecking = false
```

Comment:
Why did we change this?

Reply:
It was a bad merge, fixed.  Thanks for flagging